color.py

```diff
@@ -253,6 +253,20 @@ def fading_color(v, total):
     return _clrs[_fading_idx(v, total)]
 
 
+def get_colors(colors, col_n):
+    if colors is None:
+        colors = []
+
+    colors = colors or ([None] * col_n)
+
+    while len(colors) < col_n:
+        colors.extend(colors)
+
+    colors = colors[:col_n]
+
+    return colors
```

Comment:
What is this func for indeed?
Is it gonna be used somewhere?

Reply:
> What is this func for indeed?
> Is it gonna be used somewhere?

https://github.com/bsc-s2/pykit/blob/master/strutil/strutil.py#L409